TRIB-83: Feature/notifications page

loretdemolas opened the pull request:
Generated notifications page with routing and minor Css.

Modified home page to have a bell icon in the footer and added logic to routing.

Added placeholder text in notifications.

loretdemolas committed:
TRIB-83

Added:
notification.page.scss
-blank page, yet to run the app
notification.page.html
-Added Header 'Notifications'
notification.page.ts
-created basic component for notification page
notification.page.module.ts
-create basic Module

loretdemolas committed:
TRIB-83

Added:
notification.page.scss
-blank page, yet to run the app
notification.page.html
-Added Header 'Notifications'
notification.page.ts
-created basic component for notification page
notification.page.module.ts
-create basic Module

Signed-off-by: loretdemolas <sebast.loret@gmail.com>

loretdemolas committed:
TRIB-83

Added:
notifications.page.scss
-added some css to match homepage toolbar
notifications.page.html
-Added Header 'Notifications'
notifications.page.ts
-created basic component for notification page
notifications.page.module.ts
-create basic Module
notifications.page-routing.module.ts
-created routing to bell icon on home page
home.page.html
-added button and logic to navigate to notifications page

Signed-off-by: loretdemolas <sebast.loret@gmail.com>

haxwell commented on src/app/home/home.page.html:
```diff
@@ -18,5 +18,6 @@
     <ion-button (click)="onHomeBtnClick()" fill="clear"><ion-icon name="albums"></ion-icon></ion-button>
     <ion-button (click)="onProfileBtnClick()" fill="clear"><ion-icon name="person"></ion-icon></ion-button>
     <ion-button (click)="onSettingsBtnClick()" fill="clear"><ion-icon name="Edit Your Attributes"></ion-icon></ion-button>
+    <ion-button (click)="onGoToNotificationBtnClick()" fill="clear"><ion-icon name="notifications"></ion-icon></ion-button>
```
Eventually, we will update the footer buttons, but for now I'm not sure what all should go in there. Instead, you should add a button like the one on line 12 to launch the Notifications page.

loretdemolas replied:
Quick fix: Just committed the requested changes.

Let me know if anything else needs to be changed.

loretdemolas committed:
TRIB-83

Added:
notifications.page.scss
-added some css to match homepage toolbar
notifications.page.html
-Added Header 'Notifications'
notifications.page.ts
-created basic component for notification page
notifications.page.module.ts
-create basic Module
notifications.page-routing.module.ts
-created routing to bell icon on home page
home.page.html
-added button and logic to navigate to notifications page

Signed-off-by: loretdemolas <sebast.loret@gmail.com>

haxwell reviewed (commented):
allz good
